File: Lista02/exercício 10.py
#Dados três valores A, B e C, construa um algoritmo que escreva os valores de forma descendente (do maior para o menor).

A = float(input("Digite o primeiro número: "))
B = float(input("Digite o segundo número: "))
C = float(input("Digite o terceiro número: "))
if A > B and B > C:
    print(A, B, C)
if A > C and C > B:
    print(A, C, B)
if B > A and A > C:
    print(B, A, C)
if B > C and C > A:
    print(B, C, A)
if C > A and A > B:
    print(C, A, B)
if C > B and B > A:
    print(C, B, A)

File: Lista 13/exercício 10.c
//Escreva um algoritmo que receba três números. Elabore uma função que retorne o menor número entre os digitados.

#include <stdio.h>
#include <stdlib.h>

double menornum(double n1, double n2, double n3)
{
    double menor;
    menor = n1;
    if (n2 < menor)
    {
        menor = n2;
    }
    if (n3 < menor)
    {
        menor = n3;
    }
    return menor;
}

int main()
{
    double n1, n2, n3;

    printf("Digite o primeiro número: ");
    scanf("%lf", &n1);
    printf("Digite o segundo número: ");
    scanf("%lf", &n2);
    printf("Digite o terceiro número: ");
    scanf("%lf", &n3);

    printf("Menor = %lf\n", menornum(n1, n2, n3));

    system("PAUSE");
    return 0;
}